scripts/ci/test_index.py

```diff
@@ -144,6 +171,30 @@ def test_metadata(self):
                         "Remove these dependencies: {}".format(item['filename'], deps))
         shutil.rmtree(extensions_dir)
 
+    def test_azext_metadata(self):
+
+        historical_extensios = [
+            'keyvault-preview',
```

Comment:
Is it owned by us? Why do we ignore this one?

Reply:
Lost connected with this extension. It's not hosted in our repo and not sure whether I could remove it. So I filter it out.